### ISSUE TITLE
Wagtail Update (May 27 Staff call)

### ISSUE THREAD

**Issue by cadecairos** [MozillaFoundation/mofo-devops#755]
## Draft outline + brainstorm
1. How it was before:
20-30s: before: Already a CRM, but we did not really leverage it. We had multiple sites (ex: advocacy, teach, blogs, mozfest, buyersguide). Screenshot of mezzanine.
2. How is it now: 10s-20s One CMS to rule them all. Huge lift for dev: Change in tech (more django, less node, switch the framework) => staff had to grow new skills.
3. 30s-1m: 
    - Same tool for staff, but different site: Did you know that Mozfest and the Foundation site were actually one single site, but with two heads. It’s easy for staff to edit both of those sites since it’s the same interface and features.
      - Donate is also running the same framework, but on a different server. But, Thunderbird and MoFo donate are also like Mozfest/foundation: same website, with different entry points.
4. Highlights of major features:
    - building a website like lego: we code blocks for staff, and they organize the pages the way they want it. 
    - Preview + schedule publishing.
    - Password protected and draft pages
    - Different levels of access: full staff, fellows.
    - Easy (lol) localization \o/
5. 10s of “by the way here is the doc” (how to wagtail?) + styleguide
6. A cool new feature that no ones no about :(
    - Documents?
    - Redirections?
    - Image editing, focal area, etc
7. Next features: AI work to improve user experience + how we share how message.
8. Cool stats: traffic, number of pages, images, documentations, etc.
9. Conclusion:
    - Make sure staff feel empowered to create pages
    - Improved traffic A LOT
    - Standardization of our tools and web platform: 3 of our major web properties use the same framework.
    - Support or question:  mofo-wagtail

**Comment by sabrinang** [comment]
cc/ @sabrinang

**Comment by cadecairos** [comment]
@sabrinang I've added the draft content/ideas to the issue comment, and started a slide deck using the same template the staff call slides are using. I guess once they're ready I can copy/paste them in the staff call deck.

https://docs.google.com/presentation/d/1EMaToF2z4rrshOBLYC9Bkvdx3MEzgFZ_4zaC6bTz1cs/edit?usp=sharing

**Comment by sabrinang** [comment]
@cadecairos 
I made slides from 12-32 _(I didn't want to over write the slides you had)_: https://docs.google.com/presentation/d/1EMaToF2z4rrshOBLYC9Bkvdx3MEzgFZ_4zaC6bTz1cs/edit?usp=sharing

- integrated notes you had on github ticket + content you had on slides/presenter notes
- I left comments to highlight the sections that are missing content/copy the most ... feel free to rework any content/copy and reduce where you can _(might be a bit wordy)_
- I reordered the sections to flow from: "how did we get to wagtail (showing before/now)" -> "wagtail features" -> "results (needs a title)" -> "next steps/learn more" 

Let me know if you still need anything for Monday!

**Comment by kristinashu** [comment]
w00t awesome outline and awesome slides!

**Comment by cadecairos** [comment]
Thanks @sabrinang these look fantastic! I'll fill in the unknowns and submit them to Amanda later today.
